templates/profile/prometheus/exporter/node/defaults.sh.erb

```diff
@@ -0,0 +1,134 @@
+# Managed by puppet (nebula/profile/prometheus/exporter/defaults.sh.erb)
+<%# Set the command-line arguments to pass to the server.
+    Due to shell scaping, to pass backslashes for regexes, you need to double
+    them (\\d for \d). If running under systemd, you need to double them again
+    (\\\\d to mean \d), and escape newlines too. -%>
+ARGS="--collector.diskstats.ignored-devices=^(ram|loop|fd|(h|s|v|xv)d[a-z]|nvme\\d+n\\d+p)\\d+$ \
+      --collector.filesystem.ignored-mount-points=^/(sys|proc|dev|run)($|/) \
+      --collector.netdev.ignored-devices=^lo$ \
+      --collector.textfile.directory=/var/lib/prometheus/node-exporter"
```

Comment:
These are Debian's defaults. I converted the comments to erb comments so that the final file will be clean, but since none of us have really looked at our options yet, I figured it'd be good to leave the comments here for the moment.